scenarioo-docu-generation-example/.gitignore

```diff
@@ -1,5 +1,4 @@
 build
-out
```

Comment:
hmm, sorry, maybe my comment about this was bad, I have seen that I have this 'out' as well - so just removing it from gitignore does not realy solve the problem - either put it back or adjust that everything is built into `build` directory